### PR TITLE
feat(payment): PAYPAL-886 added container setting for spb container

### PR DESCRIPTION
**Pull request by andriiVitvitskyi1990**
#### What?
Added container setting for spb container

#### Tickets / Documentation

https://jira.bigcommerce.com/browse/PAYPAL-886

#### Screenshots
Screenshot 2021-07-20 at 11 03 36

### CODE REVIEW

**Review by phanyaa — APPROVED** [review]

**Comment by BC-tymurbiedukhin** [comment]
@andriiVitvitskyi1990 does this PR need to be merged?

**Comment by andriiVitvitskyi1990** [comment]
> @andriiVitvitskyi1990 does this PR need to be merged?

This PR is on Hold.

File: schema.json
Comment (bc-fetisov) [inline schema.json:2832]: Don't forget to remove default values for

- paymentbuttons-paypal-fundingicons
- paymentbuttons-paypal-tagline
- paymentbuttons-paypal-size

as they are not used more

Comment (andriiVitvitskyi1990) [reply schema.json:2832]: fixed

**Comment by andriiVitvitskyi1990** [comment]
> @andriiVitvitskyi1990 does this PR need to be merged?

Please merge this PR

**Comment by andriiVitvitskyi1990** [comment]
@BC-tymurbiedukhin Please merge this PR

**Review by BC-tymurbiedukhin — COMMENTED** [review]
Do we have a date when rest of translation will be provided?

**Comment by andriiVitvitskyi1990** [comment]
> Do we have a date when rest of translation will be provided?

Translation will be created after merge. Strings for translations pull from master

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

**Comment by RomanKrasinskyi** [comment]
> > Do we have a date when rest of translation will be provided?
> 
> Translation will be created after merge. Strings for translations pull from master @RomanKrasinskyi

We have to extract source phrases once per month and then push them to the smartling. After ~10 business days we get translations and push them back to repo(s) as soon as possible. Luckily next extraction will be performed within this week (today or tomorrow).

**Pull request merged**
